### ISSUE TITLE
Regarding our data set

### ISSUE THREAD

**Issue by tungpatrick** [UBC-MDS/sylvia_patrick_Titanic_Survival_ML#10]
Hey Sylvia, 

In our proposal, we said that we would be analyzing these features: Passenger classes, Sex, Age, Number of siblings/spouses onboard, Number of parents/children onboard, Fare prices. Do you think that we should "clean" our data so that we only have these features? What I mean is, do you think we should have a "cleaned_data" file where we don't have any features that we are NOT analyzing? (This is regarding both the training and testing data).

**Comment by LeeYinYing** [comment]
Hi Patrick,

I definitely think that it is a good idea to remove the data that we do not need in the clean version. I saw your pull request, thanks for cleaning the data. I will take a good look at it later.

**Comment by LeeYinYing** [comment]
I noticed that the gender_submission.csv is actually the "survived" data for the test set. Do you think we should merge that into the clean test dataset?

**Comment by LeeYinYing** [comment]
Also, for the exploratory analysis, I think we should create a clean_total dataset that includes both the test and train data. That way we can show EDA that applies to the entire sample and not just a certain set. What do you think?

**Comment by tungpatrick** [comment]
Yeah we can merge the gender_submission.csv with the test set thanks. 

In regards to the exploratory analysis, I don't think we should be applying EDA to our entire sample, just the training data.
